src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="position">
+    <form
+      action="#"
+      data-qa="big"
+      class="form form__big"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        class="big__input"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+
+    <form
+      action="#"
+      data-qa="small"
+      class="form form__small"
+    >
+      <input
+        type="text"
+        class="small__input"
+        placeholder="Try “Los Angeles“"
```

Comment:
Similar to 'big__input', the class 'small__input' should be renamed to better represent the content's meaning rather than its style.